drivers/network/netio/netio.c

```diff
@@ -0,0 +1,882 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+/* This file is (c) Johannes Thoma <johannes@johannesthoma.com 2023-2024
+   It is licensed under the GPL v2.
+ */
```

Comment:
```suggestion
/*
 * PROJECT:     NetIO drivers main file
 * LICENSE:     GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
 * PURPOSE:     A more convenient networking (TCP/IP, UDP/IP) kernel API
 * COPYRIGHT:   Copyright 2023-2024 Johannes Thoma <johannes@johannesthoma.com>
 */
```
Format all headers like that, see https://reactos.org/wiki/Coding_Style#File_Structure for reference

Reply:
Applied, thanks.